src/pages/kb/user-guide/visualizations/table-visualizations.md

```diff
@@ -29,6 +29,12 @@ You can:
 
 Redash is sensitive to the data types that are common to most databases: text, numbers, dates and booleans. But it also has special support for non-standard column types like JSON documents, images, and links.
 
+{% callout info %}
+
+Redash doesn't escape HTML markup in query results by default. So you may see odd effects if a query returns string fields that include HTML (e.g. from a web scraper). Toggle the **Allow HTML content** setting in the visualization editor to escape HTML characters.
```

Comment:
It's worth pointing out that we do sanitize it. The remaining tags are not escaped.

Reply:
Fixed 4605f25858ba965761ac09281adb099ba91cab0b